src/redux/sagas/motions/managePermissionsMotion.ts

```diff
@@ -334,6 +335,8 @@ function* managePermissionsMotion({
         state: { isRedirect: true },
       });
     }
+
+    yield clearContributorsAndRolesCache();
```

Comment:
I think this will clear the cache when the motion is created. We should do it when the motion is finalized, there are different sagas for finalizing a voting reputation motion or a multi-sig motion. :)

Reply:
Thank you for pointing this out @iamsamgibbs 🙏